src/AGPL-3.0-or-later.xml

```diff
@@ -0,0 +1,848 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="AGPL-3.0-or-later" isOsiApproved="true"
+  name="GNU Affero General Public License v3.0 or later">
+    <crossRefs>
+      <crossRef>http://www.gnu.org/licenses/agpl.txt</crossRef>
+      <crossRef>http://www.opensource.org/licenses/AGPL-3.0</crossRef>
+    </crossRefs>
+    <standardLicenseHeader>
+      Copyright (C)<alt name="copyright" match=".+">[year] [name of author]</alt>
+      This program is free software: you can redistribute it and/or modify it
+      under the terms of the GNU Affero General Public License as published
+      by the Free Software Foundation, version 3. This program is distributed
```

Comment:
Here is the text from https://www.gnu.org/licenses/gpl-3.0.en.html: 

`This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.`

Reply:
Can we have `<optional>` in standardLicenseHeader? If yes, I would put the "(or later)" as optional.

Reply:
Since this license ID is only associated with the "or-later", I think it should just be in the license header, not enclosed by optional.

BTW - optional and alt tags are allowed in the license header.

Reply:
I agree that "or later" should be in the header.
I was talking about "(at your option)" which is not always there: "either version 3 of the License, or any later version." e.g. https://bat.mpp.mpg.de/?page=license

Reply:
Ahhh - agree we should wrap the "(at your option)" in optional tags.

Reply:
Added a pull request with the optional tags - once approved, we can merge it into this PR.